_data/changelogs/docs-settings.yml

```diff
@@ -2,6 +2,12 @@ title: Settings
 type: utility
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Added color contrast checks for disabled tokens.
+    summaryAdditional:  On compilation, USWDS will test disabled element color contrast and provide a fallback color if minimum contrast is not met. If the fallback does not meet minimum requirements, USWDS will provide a warning in the terminal.
+    affectsAccessibility: true
+    affectsStyles: true
+    githubPr: 5455
```

Comment:
```suggestion
    githubPr: 5455
    githubRepo: uswds
    versionUswds: 3.8.0
```

Reply:
Ahh thank you! I think these got lost in the merge with main